proto/BUILD

```diff
@@ -59,7 +59,8 @@ bzl_library(
 )
 
 # Toolchain type provided by proto_toolchain rule and used by proto_library
-toolchain_type(
+alias(
```

Comment:
This is the only real change, everything else is just updates to get CI unstuck.